www/includes/utility.php

```diff
@@ -893,6 +879,8 @@ function entities_to_numbers($string) {
 
 function make_member_url($name, $const = '', $house = HOUSE_TYPE_COMMONS, $pid = NULL) {
 
+    # convert name to ISO so we have sensible URLs
+    $name = iconv('UTF-8', 'ISO-8859-1//TRANSLIT', $name);
```

Comment:
The issue is that in Apache the redirects break when there are accented characters because it seems to cut the URL off at the accented character and you end up with a 404. I'm failed to get this to work as there didn't seem to be a way to get the regular expressions to handle the accented characters [:alpha:] for example didn't seem to do the right thing.